common/remote/hooks/s3/s3file.cpp

```diff
@@ -376,6 +378,11 @@ unsigned __int64 S3FileReadIO::getStatistic(StatisticKind kind)
     return stats.getStatistic(kind);
 }
 
+IFile * S3FileReadIO::queryFile() const
+{
+    return file.get();
```

Comment:
trivial: get() vs operator.. see prev. comment/consistency and other similar below